### PR TITLE
pt-br translation of wallet_quickstart

### PR DESCRIPTION
**Pull request by botoserelepe**
Translated wallet_quickstart waiting for review

### CODE REVIEW

**Comment by lh1008** [comment]
Thank you @botoserelepe. Pinging @netrik182 @rating89us for review >:)

File: monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Carteiras web *(Otimizadas para conveniência)*
+
++ [mymonero.com](https://mymonero.com/)
+MyMonero é uma carteira leve direto do navegador. Está disponível também como app para iOs e como app de desktop.
+
+## Carteiras de hardware *(Otimizadas para segurança)*
+
++ [ledger.com](https://shop.ledger.com/?r=92d74dc2847a)
+Ledger é um dispositivo de altíssima segurança que armazena sua semente mnemônica.
+Não aplicável
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md]: I'd like to check with both @erciccione and @lh1008 if line 48 should read "Not Applicable" or if it's just a mistake.

File: monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
++ [trezor.io](https://trezor.io/)
+Trezor é um dispositivo alternativo de altíssima segurança que armazena sua semente mnemônica.
+
+Para uma carteira de hardware que fornece segurança extra no armazenamento e transferência de Monero mesmo se o dispositivo primário estiver com vírus, tanto o Ledger, [ledger.com](https://shop.ledger.com/?r=92d74dc2847a), quanto o Trezor, [trezor.io](https://trezor.io/), são bem reconhecidos. (Repare que o Ledger funciona agora com carteiras Monero, enquanto a compatibilidade do Trezor com essas carteiras ainda está em desenvolvimento.)
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md]:
```suggestion
Para uma carteira de hardware que fornece segurança extra no armazenamento e transferência de Monero mesmo que o dispositivo primário esteja com vírus, tanto a Ledger, [ledger.com](https://shop.ledger.com/?r=92d74dc2847a), quanto a Trezor, [trezor.io](https://trezor.io/), são bem reconhecidas. (Repare que a Ledger já funciona com carteiras Monero, enquanto a compatibilidade da Trezor com essas carteiras ainda está em desenvolvimento.)
```

File: monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Deixar que suas preferências de recursos e privacidade guiem sua escolha de carteira irá maximizar sua experiência com Monero. A riqueza do Monero em opções de carteira — do rápido aplicativo MyMonero até a oficial carteira CLI de nó completo do Monero em conjunto com o Ledger ou o Trezor — permite que o Monero agrade às diversas necessidades dos seus usuários.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md]:
```suggestion
Deixar que suas preferências de recursos e privacidade guiem sua escolha de carteira irá maximizar sua experiência com Monero. A riqueza do Monero em opções de carteira — do rápido aplicativo MyMonero até a oficial carteira CLI de nó completo, em conjunto com a Ledger ou a Trezor — permite que o Monero agrade às diversas necessidades dos seus usuários.
```

File: monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md (hunk 1 above)
Comment (erciccione) [reply monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md]: This looks like a leftover from the draft, but @lh1008 will know better than me.

Comment (lh1008) [reply monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md]: hey, yes this has to be removed from the original English file. Let me fix it. Thank you for noticing it.

Comment (lh1008) [reply monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md]: Sorry guys its ok, leave the "not applicable". This is for the site https://www.monerooutreach.org/stories/monero_wallet_quickstart.php

**Comment by lh1008** [comment]
@botoserelepe forget my last comment. No need to remove "Not applicable"

**Pull request merged**